meson.build

```diff
@@ -1371,16 +1371,19 @@ conf.set10('HAVE_BZIP2', libbzip2.found())
 libxz = dependency('liblzma',
                    required : get_option('xz'))
 conf.set10('HAVE_XZ', libxz.found())
+libxz.partial_dependency(includes: true, compile_args: true)
 
 liblz4 = dependency('liblz4',
                     version : '>= 1.3.0',
                     required : get_option('lz4'))
 conf.set10('HAVE_LZ4', liblz4.found())
+liblz4.partial_dependency(includes: true, compile_args: true)
 
 libzstd = dependency('libzstd',
                      version : '>= 1.4.0',
                      required : get_option('zstd'))
 conf.set10('HAVE_ZSTD', libzstd.found())
+libzstd.partial_dependency(includes: true, compile_args: true)
```

Comment:
you need to assign this to some name, and then use that in the libshared deps.

look at how this is done for libxenctrl/libiptc/libp11kit.

i.e. change this line to: `libzstd_cflags = libzstd.partial_dependency(includes: true, compile_args: true)`

and then use that in the `dependencies` parameter for the libbasic_compress object

Reply:
Done, also in all other meson.build files, except the one for systemd-importd which is still using liblzma directly.